### DISCUSSION TITLE
arushi-11/Analysis-of-Algorithms

### DISCUSSION THREAD

**Discussion by arushi-11** [community/community#23699] [New to GitHub]
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/arushi-11/Analysis-of-Algorithms" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/33098943/181078810-12d43303-36c7-4b0a-b5ad-81a92b29d7e0.png" class="thumbnail onebox-avatar" width="420" height="420">

<h3><a href="https://github.com/arushi-11/Analysis-of-Algorithms" target="_blank" rel="noopener nofollow ugc">arushi-11/Analysis-of-Algorithms</a></h3>

<p>Contribute to arushi-11/Analysis-of-Algorithms development by creating an account on GitHub.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>I want to remove carrisunrio as a contibutor to this repository. But don’t know how to do that. Can I get any help.<br>
Thank you.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/arushi-11">@arushi-11</a>!</p>
<p>You can view the contributors that have access to your repository under <strong>Settings</strong> &gt; <strong>Manage Access</strong>. To remove a contributor listed there, you just need to hit the trash can on the right side of the screen next to their username and then confirm in the dialog box that you want to remove them.</p>
<p>Hope that helps!</p>

**Comment by arushi-11** [comment]
<p><a class="mention" href="/that-pat">@that-pat</a> Thanks to you, for replying.<br>
But the main issue is that there is no contributors listed in Manage Access.<br>
But it still be seen in the repository.</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Did the person listed make a commit in your repository? If so, their contributions will continue to be counted.</p>

**Comment by arushi-11** [comment]
<p><a class="mention" href="/that-pat">@that-pat</a> Yes. she did.<br>
Okay! thank you for taking out your time. :slight_smile:</p>
